### PR TITLE
Clean special option keywords in filters (master)

### PR DESCRIPTION
**Pull request by timopollmeier**
When splitting a filter, some special option filter keywords
(like "first", "rows", "apply_overrides", etc.) will be changed so the
relation is changed to "=".
Also, numeric option keywords will be adjusted so their values are in
the valid range (e.g. "first" is set to 1 if it is 0 or less).

### CODE REVIEW

File: src/manage_sql.c [hunk 1]
```diff
@@ -1652,6 +1652,100 @@ parse_keyword (keyword_t* keyword)
...
+        {
+          g_free (keyword->string);
+          keyword->integer_value = 100;
+          keyword->string = g_strdup ("100");
+        }
+      keyword->relation = KEYWORD_RELATION_COLUMN_EQUAL;
+    }
+  else if (strcasecmp (keyword->column, "autofp") == 0)
+    {
+      /* min_qod must be a percentage (between 0 and 2) */
```
Comment (mattmundell) [inline src/manage_sql.c]:
```suggestion
      /* min_qod must be a either 0, 1 or 2 */
```

Comment (timopollmeier) [reply src/manage_sql.c]: Fixed the typo in 688525df01931e0d85cb343d524da8e01971c6ed

**Comment by timopollmeier** [comment]
I think having some feedback would be nice but I'd rather make it a warning message than making
the command fail.

**Comment by mattmundell** [comment]
I think sending a warning back in the GMP response is getting too complex.

Not a big deal.  GSA could even check the filter before sending the command.

**Pull request merged**